src/devices/bus/vme/vme_mvme120.h

```diff
@@ -0,0 +1,161 @@
+// license:BSD-3-Clause
+// copyright-holders:Katherine Rohl
+#ifndef MAME_BUS_VME_VME_MVME120_H
+#define MAME_BUS_VME_VME_MVME120_H
+
+#pragma once
+
+#include "emu.h"
+#include "cpu/m68000/m68000.h"
+#include "bus/vme/vme.h"
+#include "bus/rs232/rs232.h"
+#include "machine/clock.h"
+#include "machine/mc68901.h"
+
+#ifdef _MSC_VER
+#define FUNCNAME __func__
+#else
+#define FUNCNAME __PRETTY_FUNCTION__
+#endif
```

Comment:
Please don’t leak macros from header files unless absolutely necessary, particularly not when they have rather generic names like this.